src/lib/Icons.ts

```diff
@@ -206,6 +208,7 @@ type IconName = Extract<
   | 'chevron-circle-left' // previous
   | 'qrcode'
   | 'random'
+  | 'route'
```

Comment:
Het staat nu een beetje door elkaar en dat is voor nu prima.
Ik zou er wel gelukkig van worden als we de keys alfabetisch gesorteerd zouden hebben (of een andere georganiseerde volgorde), wat jij?

De comments mogen van mij ook weg hier. Die hadden we nodig voor de upgrade van ObsIdentify naar de nieuwste versie van RC-components, maar die kunnen nu weg, toch?